certification/runtime/assets.go

```diff
@@ -23,7 +28,14 @@ func imageList() []string {
 
 	i := 0
 	for _, image := range images {
```

Comment:
why are we ignoring the index, but creating an index outside of the loop an incrementing? can't we have i declared here and not increment? I might be missing something though.

Reply:
We're ignoring the key, not the index. images is a map.

Reply:
That it is, was thinking slice/array for some reason.